javascript/pad_zeros.js

```diff
@@ -0,0 +1 @@
+function pad(v){return ('0'+v).substr(-2);} //so for example pad(8) = 08.
```

Comment:
Is this code only supposed to work for single and double digit ints?
pad(3.3) produces '.3' and pad(1000) produces '00'.

Reply:
Ah, great!